backend/src/main/java/de/roman/toolio/model/InquiryPart.java

```diff
@@ -0,0 +1,24 @@
+package de.roman.toolio.model;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Document(collection="inquiryParts")
+public class InquiryPart {
+    // Dimensions
```

Comment:
your collection currently does not include an ID, MongoDB will assign a default ID

Reply:
Thanks! I will be using annotation ```@Id``` in order to lock a variable to my ```_id```!